client/service/src/lib.rs

```diff
@@ -121,7 +121,10 @@ where
 		overseer_handle.clone(),
 		// We want that collators wait at maximum the relay chain slot duration before starting
 		// to recover blocks.
-		RecoveryDelay { min: core::time::Duration::ZERO, max: relay_chain_slot_duration },
+		RecoveryDelay {
+			min: relay_chain_slot_duration.checked_div(2).expect("2 is larger than 0; qed"),
```

Comment:
```suggestion
			min: relay_chain_slot_duration / 2,
```

Reply:
And some comment here why we are doing this, something like "we want to wait for half to the next slot to have something available bla". Basically what we talked about.